lib/chewy/fields/base.rb

```diff
@@ -53,6 +55,18 @@ def compose(*objects)
 
     private
 
+      def field_with_ignore_blank_flag?(result)
+        @options[:ignore_blank] == true && field_with_empty_value?(result)
+      end
+
+      def geo_point_field_without_ignore_blank_flag?(result)
+        !@options[:ignore_blank] && @options[:type] == :geo_point && field_with_empty_value?(result)
+      end
+
+      def field_with_empty_value?(result)
+        !result.nil? && result.empty?
+      end
```

Comment:
You can remove this method, since [blank? method](https://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark) from active support will behave exactly as you are expecting